user-docs/data-format.md

```diff
@@ -57,11 +67,11 @@ We use a normalized coordinate system with the origin `0,0` at the bottom left a
   In some rare cases we use the image coordinate system. This is mainly for pixel access of the image arrays. Here a unit is one pixel, origin is "top left" and "bottom right" is the maximum x,y.
 
 #### Timestamps
-All indexed data, (for example, still frames from the world camera, still frames from the eye camera, gaze and pupil coordinates, and so on) has timestamps associated to for synchronization purposes. The timestamp is derived from `CLOCK_MONOTONIC` on Linux and MacOS. 
+All indexed data, (for example, still frames from the world camera, still frames from the eye camera, gaze and pupil coordinates, and so on) has timestamps associated to for synchronization purposes. The timestamp is derived from `CLOCK_MONOTONIC` on Linux and MacOS.
```

Comment:
Edit: 

All indexed data - still frames from the world camera, still frames from the eye camera(s), gaze coordinate, and pupil coordinates, etc. - have timestamps associated for synchronization purposes.